pkg/commands/componentarchive/remote/remote_test.go

```diff
@@ -135,4 +135,75 @@ var _ = Describe("Remote", func() {
 		Expect(showOpts.Run(ctx, logr.Discard(), testdataFs)).To(HaveOccurred())
 	})
 
+	It("should copy a component descriptor and its blobs from the source repository to the target repository.", func() {
```

Comment:
i thought that we wanted to test that also local blobs are copied but the component descriptor does not contain a local blob or least we do not test if the blob is copied?